base-notebook/apt.txt

```diff
@@ -0,0 +1,19 @@
+build-essential
+curl
+dfc
+file
+git
+htop
+jq
+less
+ncdu
+ranger
+rclone
+rsync
+silversearcher-ag
+software-properties-common
+tig
+tmux
+tree
+unzip
+vim
```

Comment:
Are all this dependencies really needed for this particular change? I feel some are just here for adding tools, but could be in another PR?